packages/react-components/react-layout/src/Container/Container.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import camelCase from 'lodash.camelcase';
+import { ForwardRefComponent } from '@project44-manifest/react-types';
+import { useStyles } from './Container.styles';
+import { ContainerElement, ContainerProps } from './Container.types';
+
+export const Container = React.forwardRef((props, forwardedRef) => {
```

Comment:
what's the difference between this and a `Box`? Is there any best practices when to use one over the other?

Reply:
This is a simple component for creating centered layouts: 

https://www.manifestdesignsystem.com/docs/components/container